android/quest/src/main/res/values/strings.xml

```diff
@@ -109,6 +109,11 @@
     <string name="extraction_in_progress">Processing questionnaire data…</string>
     <string name="loading_questionnaire">Loading questionnaire…</string>
     <string name="clear_all">Clear All</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+
+    <string name="location_permissions_denied">Location Access Denied: To use this feature, please enable location permissions in your device settings.</string>
```

Comment:
Would it be better to describe the feature that requires this permission? @dubdabasoduba for better user experience?